src/jsc/jsc_value.hpp

```diff
@@ -125,12 +125,12 @@ inline bool jsc::Value::is_valid(const JSValueRef &value) {
 
 template<>
 inline bool jsc::Value::is_decimal128(JSContextRef ctx, const JSValueRef &value) {
-    return true; // FIXME: must be an instance of Decimal128 from BSON
+    return JSValueIsObject(ctx, value); // FIXME: can we do better?
```

Comment:
if it is `Decimal128` it will have the property named `_bsontype =  "Decimal128"`

Reply:
Perhaps https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/isPrototypeOf could be useful here?